Ex3/GarageLogic/VehiclesDictionary.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GarageLogic
+{
+    public class VehiclesDictionary
```

Comment:
Looks good, But I think it better to give a more valuable name like Vehicle Default configuration